draft-ietf-quic-transport.md

```diff
@@ -743,14 +743,25 @@ have special rules for populating the packet number field.
 
 ### Initial Packet Number {#initial-packet-number}
 
-The initial value for packet number MUST be selected from an uniform random
-distribution between 0 and 2^31-1.  That is, the lower 31 bits of the packet
-number are randomized.  {{?RFC4086}} provides guidance on the generation of
-random values.
-
-The first set of packets sent by an endpoint MUST include the low 32-bits of the
-packet number.  Once any packet has been acknowledged, subsequent packets can
-use a shorter packet number encoding.
+The initial value for packet number MUST be selected randomly from a range
+between 0 and 2^32 - 1024.  This value is selected so that Initial and Handshake
```

Comment:
Is this inclusive of 0 and 2 ^ 32 - 1024? I would say yes, and I don't actually care, but the spec should say it.

Reply:
I wanted to avoid invoking the spectre of #832, but I will clarify (the easiest is probably [0, 2^32-1024] because that makes a check using a bitmask easier).